### PR TITLE
Added validation before package confirm

### LINKED ISSUE #153: Confirm view: Add error or informative message when "Confirmar Recepcion" button fails
### Explication

In confirm view (https://dev--azordev-dasher-user.netlify.app/confirm/902b7f07-da34-4ab5-9a8c-d9db03ba55b5) when user clicks confirm button and the package is already confirmed before, any action occurs, please add one message error or informative to say this to the user

### Screenshots

imagen

### Checklist

This issue can be closed when the following tasks are complete:

- [ ] When package can't be confirmed, please show a message explaining what is the problem

### PR DESCRIPTION
**Pull request by e-azocar**
### Description

I added some validations to show a message when the user tries to confirm a package already confirmed.

fixes #153 

#### Screenshots
This modal only is show when the package is already confirmed previously
Captura de pantalla 2022-01-10 192953

### CODE REVIEW

File: src/pages/Chat/Chat.styled.jsx [hunk 1]
```diff
@@ -133,8 +133,8 @@ export const MessageBox = styled.div`
 `
 
 export const Avatar = styled.img`
+float: ${({ type }) => (type === 'client' ? `left` : `right`)};
```
Comment (Israel-Laguan) [inline src/pages/Chat/Chat.styled.jsx]: Please mind the indentation

File: src/pages/Confirm/index.jsx [hunk 2]
```diff
@@ -49,7 +53,9 @@ const Confirm = () => {
   /** @param {React.FormEvent<HTMLFormElement>} event */
   const submitConfirmation = async event => {
     if (event.name && event.RUT && event.phone) {
-      confirmPackage(event, id)
+      await confirmPackage(event, id)
+      console.log(packageInformation)
+      console.log(loading)
```
Comment (Israel-Laguan) [inline src/pages/Confirm/index.jsx]: Remove these please

File: src/pages/Confirm/index.jsx [hunk 3]
```diff
@@ -17,24 +17,28 @@ const Confirm = () => {
...
-        toggleErrorModal(true)
-      } else {
-        toggleRatingModal(true)
+      if (orderStatus === 'ready' || orderStatus === 'collected' || orderStatus === 'in_travel') {
+        return toggleRatingModal(true)
+      } else if (
+        orderStatus === 'rated' ||
+        orderStatus === 'delivery_confirmed' ||
+        orderStatus === 'delivery_rejected'
+      ) {
```
Comment (Israel-Laguan) [inline src/pages/Confirm/index.jsx]: can you try something like

```js
const incompleteStatus = ['ready', 'collected', 'in_travel'];

if (incompleteStatus.includes(orderStatus)) {

```

**Review by heliomar-pena — APPROVED** [review]
I see good the code. But i don't know how I can test the page
imagen

File: src/pages/Confirm/index.jsx [hunk 4]
```diff
@@ -17,24 +17,27 @@ const Confirm = () => {
...
-      if (orderStatus === 'rated' || orderStatus === 'delivery_confirmed' || orderStatus === 'delivery_rejected') {
-        console.log(orderStatus)
-        toggleErrorModal(true)
-      } else {
-        toggleRatingModal(true)
+      if (incompleteStatus.includes(orderStatus)) {
+        return toggleRatingModal(true)
+      } else if (completeStatus.includes(orderStatus)) {
+        return toggleErrorModal(true)
       }
```
Comment (heliomar-pena) [inline src/pages/Confirm/index.jsx]: Add validation "El paquete y el RUT no concuerdan"

**Review by heliomar-pena — CHANGES_REQUESTED** [review]
A little change

**Review by Israel-Laguan — APPROVED** [review]

**Review by heliomar-pena — APPROVED** [review]
I see it good

image

image

But I think that at the first error (El RUT y el Id del paquete no coinciden, o el paquete no existe), the user doesn't should be redirected. Because what if is a common client that wrote wrong the RUT, and the page redirect he to other page and don't know how to come back to the confirm page?

Or what do you think @angelik0828

**Review by heliomar-pena — APPROVED** [review]
Thanks for the changes

**Pull request merged**